lib/ditto.ts

```diff
@@ -58,19 +78,34 @@ const setupCommands = () => {
     const cmd = program
       .command(commandConfig.name)
       .description(commandConfig.description)
-      .action((str, options) => executeCommand(commandConfig.name, options));
+      .action((options) => {
+        return executeCommand(commandConfig.name, options);
+      });
 
-    if ("commands" in commandConfig) {
+    if (commandConfig.flags) {
+      Object.entries(commandConfig.flags).forEach(
+        ([flags, { description, processor }]) => {
+          if (processor) {
+            cmd.option(flags, description, processor);
+          } else {
+            cmd.option(flags, description);
+          }
```

Comment:
nit: I think this is the same as just
```suggestion
cmd.option(flags, description, processor)
```